packages/otp2-tile-overlay/src/util.ts

```diff
@@ -29,6 +29,26 @@ export const generateLayerPaint = (color?: string): any => {
       "circle-stroke-color": "#333",
       "circle-stroke-width": 3
     },
+    areaStops: {
+      "circle-color": [
```

Comment:
This is future proofing for if OTP ever sends that data. For now, it does nothing. But because of how annoying otp-ui updates are I'd prefer not to have to make a patch for when the new feature comes